sonar-plugin/org.wso2.code.quality.sonar.plugin/dependency-reduced-pom.xml

```diff
@@ -0,0 +1,218 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">

```

Comment:
Missing the license header. Fix in all other places as well

Reply:
Deleted dependency-reduced-pom.xml as its auto generated